website/docs/specification/DigitalFacilityRecord.md

```diff
@@ -7,15 +7,15 @@ import Disclaimer from '../\_disclaimer.mdx';
 
 <Disclaimer />
 
-* DFR context : [JSON-LD @context](https://test.uncefact.org/vocabulary/untp/dfr/0/untp-dfr-context-0.3.9.jsonld)
-* DFR schema : [JSON schema](https://test.uncefact.org/vocabulary/untp/dfr/0/untp-dfr-schema-0.3.9.json)
-* DFR sample : [JSON Sample](../../samples/untp-digital-facility-record-v0.3.9.json)
+* **[Latest JSON-LD @context](https://test.uncefact.org/vocabulary/untp/dfr/0/untp-dfr-context-0.3.10.jsonld)**
+* **[Latest JSON schema](https://test.uncefact.org/vocabulary/untp/dfr/0/untp-dfr-schema-0.3.10.json)**
+* **[Sample instance](../../samples/untp-digital-facility-record-v0.3.10.json)**
 
 ## Versions
 
 | DPP Version | Date | status  | Change summary | JSON-LD Context  | JSON Schema  |
```

Comment:
```suggestion
| DFR Version | Date | status  | Change summary | JSON-LD Context  | JSON Schema  |
```

Reply:
^ if that isn't a great example of stumbling over our own acronym.